### PR TITLE
Seasrainf improve

### PR DESCRIPTION
**Pull request by Tinkaa**
Few changes in config parameters for the seasonal forecasts, and adding the exploratory notebook on aggregation of the forecasts and used for discussion with IRI

### CODE REVIEW

File: indicators/drought/config.py [hunk 1]
```diff
@@ -57,29 +57,22 @@ def parameters(self, country):
...
-    # IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Oct%202020)/(Nov%202020)/RANGEEDGES/data.nc"
     IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Mar%202017)/(Jan%20{NEXT_YEAR})/RANGEEDGES/data.nc"
-    # IRI_URL = f"https://iridl.ldeo.columbia.edu/SOURCES/.IRI/.FD/.NMME_Seasonal_Forecast/.Precipitation_ELR/.prob/F/(Jan%202020)/(Jan%20{NEXT_YEAR})/RANGEEDGES/C/(Below_Normal)/RANGEEDGES/data.nc"
-
     IRI_DIR = "IRI"
     #currently it is solely downloading belowaverage data, this can be changed but will make downloading times longer.
-    IRI_NC_FILENAME_RAW = f"IRI_2020{NEXT_YEAR}.nc"
-    IRI_NC_FILENAME_CRS = f"IRI_2020{NEXT_YEAR}_crs.nc"
+    IRI_NC_FILENAME_RAW = f"IRI_2017{TODAY_YEAR}.nc"
+    IRI_NC_FILENAME_CRS = f"IRI_2017{TODAY_YEAR}_crs.nc"
```
Comment (joseepoirier) [inline indicators/drought/config.py:64]: can we add a comment explaining why we moved away from the 2020 dataset?
also, did you mean to use NEXT_YEAR on lines 63-64?

File: indicators/drought/config.py [hunk 2]
```diff
@@ -57,29 +57,22 @@ def parameters(self, country):
...
 
 
     ### ICPAC
-    #TODO: probably want to download directly from the ftp server instead of uploading to GDrive and downloading from there. But couldn't figure out how to do that
     ICPAC_GDRIVE_ZIPID = "13VQTVj5Lwm6jHYBMIf7oUZt5k1-alwjf"
     ICPAC_DIR = "icpac"
     #Raw data can be processed for all dates, for the one with crs, we only want to one of interest
     ICPAC_PROBFORECAST_REGEX_RAW = "ForecastProb*.nc"
-    ICPAC_PROBFORECAST_REGEX_CRS = "ForecastProb*{month}{year}_crs.nc"
+    ICPAC_PROBFORECAST_REGEX_CRS = "ForecastProb*{month}{year}_{tercile}_crs.nc"
```
Comment (joseepoirier) [inline indicators/drought/config.py:75]: what is _{tercile} referring to here? let's add a comment

**Review by joseepoirier — COMMENTED** [review]
I would hold off pushing the eth_rainfall_aggregate code since we have a more structured and informed way of proceeding now that we received input from IRI. what do you think?

**Comment by Tinkaa** [comment]
> I would hold off pushing the eth_rainfall_aggregate code since we have a more structured and informed way of proceeding now that we received input from IRI. what do you think?

Hmm fair point. Think that works for now, though would say it is okay to leave it public, just on a separate branch. Do you agree?

**Comment by Tinkaa** [comment]
Created a separate branch seasrainf-aggr that includes the eth_rainfall_aggregate code, to prevent it being on main while still being in experimental stage.

**Pull request merged**